src/classiceditor.js

```diff
@@ -105,78 +108,81 @@ export default class ClassicEditor extends Editor {
 	 *				console.error( err.stack );
 	 *			} );
 	 *
-	 * Creating an instance when using CKEditor from source (make sure to specify the list of plugins to load and the toolbar):
+	 * This is the most convenient and common way to initialize the editor. The element's content will be used as the editor data.
```

Comment:
```suggestion
	 * The element's content will be used as the editor data.
```

This is not necessarily the most convenient way to initialize the editor. Also, this sentence does not help much. It is just a statistics ;)

Reply:
We give somebody two (and a half) ways to init editor. I wanted to write which is more common, so probably better for them.

But yeah it is not necessary.